xblock/django/request.py

```diff
@@ -20,7 +27,7 @@ def webob_to_django_response(webob_response):
     return django_response
 
 
-class HeaderDict(MutableMapping):
+class HeaderDict(MutableMapping, six.Iterator):
```

Comment:
It lets you write a class with `__next__()`, and have it be an iterator on python 2 (having a `next()` method.  But on python3 it doesn't add the `next()` method. https://pythonhosted.org/six/#six.Iterator